openvpn/js/view/Server.js

```diff
@@ -91,6 +91,15 @@ Ext.define('Ung.apps.openvpn.view.Server', {
                 value: '{settings.authUserPass}',
                 disabled: '{settings.serverEnabled == false}'
             },
+        },            {
+            xtype: 'checkbox',
+            boxLabel: 'Add TOTP prompt keyword to client config',
```

Comment:
is there a better user identifiable string for this label? "Add TOTP Configuration" or something?